main.cpp

```diff
@@ -107,15 +125,20 @@ int main()
 	std::stringstream failed_combo_names("ARG! DAMMIT! DARN! NOO!");
 	const std::vector<std::string> failed_combos = initialize_combolist(failed_combo_names);
 
+	std::vector<std::string> wordlist;
+	initialize_wordlist("words.txt", wordlist);
+
 	// Word to be guessed
-	const std::string_view word = "Oxygen";
+	const size_t random_word_index = randomRange(0, wordlist.size());
+	const std::string random_word = wordlist[random_word_index];
```

Comment:
Don't copy strings so much...

Reply:
"Make sure no parameter expression ever depends on another parameter expression in the same call That's undefined behavior"

Came to mind but guess I didn't use it properly or misunderstood it

Reply:
Yeah that's not particularly applicable here; you have an array of `std::string` objects, you just need to refer to one of them to work with it, don't need to copy it.
```cpp
const std::string& random_word = word_list.at(random_word_idx);
```